test/general/lambda_naming.pass.cpp

```diff
@@ -58,7 +60,13 @@ int main() {
     EXPECT_TRUE(!is_equal, "wrong return value from equal");
     auto does_1_exist = ::std::find(policy, buf_begin, buf_end, 1);
     EXPECT_TRUE(does_1_exist - buf_begin == 1000, "wrong return value from find");
-#endif
+#endif // !_ONEDPL_FPGA_DEVICE
+
+#elif !__SYCL_UNNAMED_LAMBDA__ && _ONEDPL_BACKEND_SYCL
+    // ::std::for_each(policy, buf_begin, buf_end, [](int& x) { x++; }); // It's not allowed. Policy with different name is needed
```

Comment:
```suggestion
    // TODO: move it to a negative test once it is supported.
    // ::std::for_each(policy, buf_begin, buf_end, [](int& x) { x++; }); // It's not allowed. Policy with different name is needed
```

Reply:
Do we plan negative tests support? Do you know anything about it?

Reply:
Sorry, I do not know. The case you provided may be a motivation to enable it.